### PR TITLE
document --include (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by rjeschmi** [comment]
This is a great feature and has a bunch of different uses. Definitely we want to introduce the simplest uses first (overriding easyblocks and naming schemes) without confusing the reader

**Comment by boegel** [comment]
@rjeschmi: completed this, up for giving this another look?

**Comment by rjeschmi** [comment]
It looks fine, it is a pretty big feature. It will take some time to understand problems people have with it.

File: docs/Including_additional_Python_modules.rst [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+
+* :ref:`include_easyblocks`
+* :ref:`include_mns`
+* :ref:`include_toolchains`
+
+
+.. _include_general_remarks:
+
+General aspects of ``--include`` options
```
Comment (rjeschmi) [inline docs/Including_additional_Python_modules.rst]: this should maybe be --include-* just to avoid confusion over a generic --include option.

Comment (fgeorgatos) [inline docs/Including_additional_Python_modules.rst]: +1

**Comment by boegel** [comment]
https://github.com/hpcugent/easybuild-framework/pull/1301 is merged in, so this is good to go as well.

Thanks @rjeschmi and @fgeorgatos for the feedback here!

**Pull request merged**